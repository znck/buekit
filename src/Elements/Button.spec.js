import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'

import xButton from './Button.vue'

describe('Button.vue', () => {
	it('matches snapshot', () => {
		const wrapper = shallow(xButton, {})
		const renderer = createRenderer()

		renderer.renderToString(wrapper.vm, (err, str) => {
			if (err) {
				throw new Error(err)
			}
			expect(str).toMatchSnapshot()
		})
	})
})
