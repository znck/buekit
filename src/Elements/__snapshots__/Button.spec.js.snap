// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.vue matches snapshot 1`] = `<button class="button"></button>`;
